modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminContent.tsx

```diff
@@ -240,21 +238,32 @@ const TranslationAdminContent = ({
 			/>
 		</>
 	);
-};
+}
 
-TranslationAdminContent.propTypes = {
-	activeLanguageIds: PropTypes.arrayOf(PropTypes.string),
-	ariaLabels: PropTypes.shape({
-		default: PropTypes.string,
-		manageTranslations: PropTypes.string,
-		managementToolbar: PropTypes.string,
-		notTranslated: PropTypes.string,
-		tranlated: PropTypes.string,
-	}),
-	availableLocales: PropTypes.arrayOf(PropTypes.object).isRequired,
-	defaultLanguageId: PropTypes.string.isRequired,
-	lastDeletedLocale: PropTypes.object,
-	translations: PropTypes.object,
-};
+interface IProps extends Translations {
+	onAddLocale?: (localeId: string) => void;
+	onCancel?: React.MouseEventHandler<HTMLButtonElement>;
+	onDone?: React.MouseEventHandler<HTMLButtonElement>;
+	onRemoveLocale?: (localeId: string) => void;
+}
 
-export default TranslationAdminContent;
+export interface Locale {
+	displayName: string;
+	id: string;
+	label: string;
+	symbol: string;
+}
+
+export interface Translations {
+	activeLanguageIds?: string[];
+	ariaLabels?: {
+		default?: string;
+		manageTranslations?: string;
+		managementToolbar?: string;
+		notTranslated?: string;
+		translated?: string;
+	};
+	availableLocales: Locale[];
+	defaultLanguageId: string;
+	translations?: {[key: string]: unknown};
+}
```

Comment:
These should also be moved above IProps since that interface references Translations